Remove viz docs and datasets from framework docs (#3283)
kedro-org/kedro: docs/source/get_started/new_project.md

## Visualise a Kedro project

> This section swiftly introduces project visualisation using Kedro-Viz. See the {doc}`visualisation documentation <kedro-viz:kedro-viz_visualisation> for more detail.

Author: ```suggestion
This section swiftly introduces project visualisation using Kedro-Viz. See the {doc}`visualisation documentation<kedro-viz:kedro-viz_visualisation> for more detail.
```

* Learn hands-on: If you prefer to learn hands-on, move on to the [spaceflights tutorial](../tutorial/spaceflights_tutorial.md). The tutorial illustrates how to set up a working project, add dependencies, create nodes, register pipelines, set up the Data Catalog, add documentation, and package the project.

> * How-to guides: The documentation sections following the tutorial explain {doc}`how to visualise a Kedro project <kedro-viz:kedro-viz_visualisation>` and [how to combine Kedro with a Jupyter notebook](../notebooks_and_ipython/kedro_and_notebooks.md).

Author: ```suggestion
* How-to guide for notebook users: The documentation section following the tutorial explains [how to combine Kedro with a Jupyter notebook](../notebooks_and_ipython/kedro_and_notebooks.md).
```
